core/src/com/geekbrains/td/BulletEmitter.java

```diff
@@ -27,11 +28,19 @@ public BulletEmitter(GameScreen gameScreen) {
         this.tmp = new Vector2(0, 0);
         this.templates = new HashMap<String, BulletTemplate>();
```

Comment:
Starting from Java 1.6 generic diamonds can be written like this: `HashMap<String, BulletTemplate> templates = new HashMap<>()`.